sound/soc/sof/intel/hda-stream.c

```diff
@@ -363,12 +363,30 @@ int hda_dsp_stream_hw_params(struct snd_sof_dev *sdev,
 			  sd_offset + SOF_HDA_ADSP_REG_CL_SD_CBL,
 			  hstream->bufsize);
 
```

Comment:
The commit message is quite good :-)
I would like the sequence to be inserted in the code as comments so that we have a trace of the hardware work-around even after this patch is squashed. 
Thanks!

Reply:
yes, I will embed the sequence as comments in our code.